docker/kafka-ui-zookeeper-ssl.yml

```diff
@@ -24,12 +24,12 @@ services:
       KAFKA_CLUSTERS_0_SCHEMAREGISTRY: http://schemaregistry0:8085
       KAFKA_CLUSTERS_0_KAFKACONNECT_0_NAME: first
       KAFKA_CLUSTERS_0_KAFKACONNECT_0_ADDRESS: http://kafka-connect0:8083
-      ZOOKEEPER_CLIENTCNXNSOCKET: org.apache.zookeeper.ClientCnxnSocketNetty
-      ZOOKEEPER_CLIENT_SECURE: 'true'
-      ZOOKEEPER_SSL_KEYSTORE_LOCATION: /etc/kafka/secrets/kafka.zookeeper.keystore.jks
-      ZOOKEEPER_SSL_KEYSTORE_PASSWORD: 12345678
-      ZOOKEEPER_SSL_TRUSTSTORE_LOCATION: /etc/kafka/secrets/kafka.zookeeper.truststore.jks
-      ZOOKEEPER_SSL_TRUSTSTORE_PASSWORD: 12345678
+      ZOOKEEPER_CLUSTERS_0_CLIENTCNXNSOCKET: org.apache.zookeeper.ClientCnxnSocketNetty
```

Comment:
Could we use same params structure like: 
```
KAFKA_CLUSTERS_0_ZOOKEEPER_PROPERTIES_CLIENTCNXNSOCKET
KAFKA_CLUSTERS_0_ZOOKEEPER_PROPERTIES_SECURE
...
```

Reply:
Done. And it works anyway, I dunno how it picks it up.

```14:41:35.603 [kafka-admin-client-thread | adminclient-1] DEBUG com.provectus.kafka.ui.service.ZookeeperService - Start getting Zookeeper metrics for kafkaCluster: local```